File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/07 - Nested Conditional Statements Exercise/08. Hotel Room .py
month = input()
nights = int(input())

studio = 0
apartment = 0

if month == "May" or month == "October":
    studio = 50 * nights
    apartment = 65 * nights
    if 7 < nights <= 14:
        studio -= studio * 0.05
    elif nights > 14:
        studio -= studio * 0.3
        apartment -= apartment * 0.1
elif month == "June" or month == "September":
    studio = 75.20 * nights
    apartment = 68.70 * nights
    if nights > 14:
        studio -= studio * 0.2
        apartment -= apartment * 0.1
elif month == "July" or month == "August":
    studio = 76 * nights
    apartment = 77 * nights
    if nights > 14:
        apartment -= apartment * 0.1

print(f"Apartment: {apartment:.2f} lv.")
print(f"Studio: {studio:.2f} lv.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/10. УПРАЖНЕНИЕ ПО-СЛОЖНИ ПРОВЕРКИ/08. Hotel Room.js
function HotelRoom(...input) {
    let month = input.shift();
    let nights = Number(input.shift());

    let studio = 0;
    let apartament = 0;

    if (month === "May" || month === "October") {
        studio = 50;
        apartament = 65;
        if (nights > 7 && nights <= 14) {
            studio *= 0.95;
        } else if (nights > 14) {
            studio *= 0.7;
        }
    } else if (month === "June" || month === "September") {
        studio = 75.20;
        apartament = 68.70;
        if (nights > 14) {
            studio *= 0.8;
        }
    } else if (month === "July" || month === "August") {
        studio = 76;
        apartament = 77;
    }
    if (nights > 14) {
        apartament *= 0.9;
    }

    console.log(`Apartment: ${(apartament * nights).toFixed(2)} lv.`)
    console.log(`Studio: ${(studio * nights).toFixed(2)} lv.`)
}

HotelRoom(['May', '15']);  // Expected Output:
// Apartment: 877.50 lv.
// Studio: 525.00 lv.

HotelRoom(['June', '14']);  // Expected Output:
// Apartment: 961.80 lv.
// Studio: 1052.80 lv.

HotelRoom(['August', '20']);  // Expected Output:
// Apartment: 1386.00 lv.
// Studio: 1520.00 lv.
